helm/linkerd-bundle/values.yaml

```diff
@@ -31,8 +31,6 @@ apps:
     enabled: true
     namespace: linkerd
     namespaceConfig:
-      annotations:
-        linkerd.io/inject: disabled
```

Comment:
please don't delete. it's recommended by linkerd (see https://github.com/linkerd/linkerd2/blob/main/charts/linkerd-control-plane/templates/namespace.yaml#L11)